assets/node.yaml

```diff
@@ -111,9 +112,13 @@ spec:
             type: DirectoryOrCreate
           name: csi-volumes-map
         - emptyDir:
-            # this tells Kubernetes to mount a tmpfs (RAM-backed filesystem)
             medium: Memory
           name: csi-data-dir
+        - hostPath:
+          # hostPath under /run are mounted as a tmpfs (RAM-backed filesystem)
+            path: /run/csi-data-dir
+            type: DirectoryOrCreate
+          name: csi-data-dir
```

Comment:
suspect this is what introduced the duplicate

Reply:
A rebase and manual merge issue for this file, as it has changed directory.